Assets/LeapMotion/Modules/GraphicRenderer/Scripts/LeapGraphic.cs

```diff
@@ -290,6 +290,13 @@ public abstract partial class LeapGraphic : MonoBehaviour, ISpaceComponent, ISer
       }
     }
 
+    /// <summary>
+    /// Called by graphic groups when a renderer's attached space changes.
+    /// </summary>
+    public void SetAnchor(LeapSpaceAnchor anchor) {
```

Comment:
I'd recommend changing the name of this method to something like `OnUpdateAnchor`, to make it more clear that it's not something that should be called by the user.

Reply:
Yeah, can do. Happy to modify the pathway here further if you'd like, this was quick-and-dirty as a solution.